assets/src/edit-story/components/panels/textStyle/textStyle.js

```diff
@@ -128,10 +128,12 @@ function StylePanel({ selectedElements, pushUpdate }) {
         />
         <ToggleButton
           icon={<BoldIcon />}
-          value={bold === true}
+          value={fontWeight === 700}
           iconWidth={9}
           iconHeight={10}
-          onChange={(value) => pushUpdate({ bold: value }, true)}
+          onChange={(value) => {
+            pushUpdate({ fontWeight: value ? 700 : 400 }, true);
```

Comment:
Again, if font only has 800 or 900 rather than 700, I think this should switch to that.